pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/route/UserRegistrationRoute.java

```diff
@@ -177,8 +177,10 @@ public Object handle(Request request, Response response, UserRegistrationPayload
                 var pair = signUpNewOperator(response, handle, study, auth0UserId.get(),
                         payload, clientConfig, mgmtClient);
                 operatorUser = pair.getOperatorUser();
-                triggerUserRegisteredEvents(handle, study, operatorUser, pair.getParticipantUser());
-                publishRegisteredPubSubMessage(studyGuid, pair.getParticipantUser().getGuid());
+                if (!payload.skipTriggerEvents()) {
+                    triggerUserRegisteredEvents(handle, study, operatorUser, pair.getParticipantUser());
+                    publishRegisteredPubSubMessage(studyGuid, pair.getParticipantUser().getGuid());
+                }
```

Comment:
Might be worth a log.warn in an else block, just so we know when these events are skipped, as a backup in case something gets to prod and it shouldn't.